Userland/Utilities/dd.cpp

```diff
@@ -234,9 +234,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         }
     }
 
-    if (status == Default) {
+    if (status != None) {
         warnln("{}+{} blocks in", total_blocks_in, partial_blocks_in);
         warnln("{}+{} blocks out", total_blocks_out, partial_blocks_out);
+    }
+    if (status != Noxfer) {
```

Comment:
```
'none' suppresses everything but error messages, 'noxfer' suppresses the final transfer stats..
```
But this change will print the transfer statistics for `status=None` right as your `status != Noxfer` is true?

Reply:
Before the patches:
![image](https://github.com/SerenityOS/serenity/assets/33182938/1f424786-ebd9-45c7-9862-ef718d28e17b)

After the patches (which is not the correct behavior):
![image](https://github.com/SerenityOS/serenity/assets/33182938/ee5a4b40-fc95-4705-8d3b-875179203e01)